README.rst

```diff
@@ -5,9 +5,108 @@ Python language bindings for ev3dev
     :target: https://travis-ci.org/rhempel/ev3dev-lang-python
 .. image:: https://readthedocs.org/projects/python-ev3dev/badge/?version=latest
     :target: http://python-ev3dev.readthedocs.org/en/latest/?badge=latest
-    :alt: Documentation Statu
+    :alt: Documentation Status
 
-This is a python library implementing unified interface for ev3dev_ devices.
+A Python library implementing unified interface for ev3dev_ devices.
 
-.. _ev3dev: http://ev3dev.org
+Resources
+---------
+
+`Getting Started with ev3dev <ev3dev-getting-started_>`_
```

Comment:
This is a defect in GitHub's handling of this style of linking:

```
`Getting Started with ev3dev <ev3dev-getting-started_>`_
...
.. _ev3dev-getting-started: http://www.ev3dev.org/docs/getting-started/
```
I'll open a defect with GitHub